### PR TITLE
Feature/20221104 faq section

### PR DESCRIPTION
**Pull request by NRaeewal**

### CODE REVIEW

**Comment by NRaeewal** [comment]
[Frequently asked questions](https://trello.com/c/NF9Wntnf/35-frequently-asked-questions)

File: app/controllers/frequently_asked_questions_controller.rb [hunk 1]
```diff
@@ -1,44 +0,0 @@
...
-    @title = 'FAQ'
-    @meta_description = "Answers to some commonly asked questions when dealing with a custom software company and dealing with BiteSite specifically."
-    @frequently_asked_questions = @frequently_asked_questions.in_order
-  end
-
-  def show
-  end
-
-  def new
-  end
```
Comment (AnnaBaranova) [inline app/controllers/frequently_asked_questions_controller.rb:14]: @NRaeewal I think it's better to transfer all CRUD functionality to a new controller and re-use existing rails forms to create/update/delete questions. I added a method to check a user role (window.is...), so you can show links for adding new questions or updating/deleting somewhere in UI to keep the ability to add questions from UI.

File: app/controllers/frequently_asked_questions_controller.rb [hunk 2]
```diff
@@ -1,44 +1,56 @@
 class FrequentlyAskedQuestionsController < ApplicationController
-  load_and_authorize_resource
+  # load_and_authorize_resource
```
Comment (AnnaBaranova) [inline app/controllers/frequently_asked_questions_controller.rb:2]: @NRaeewal do you want to keep this comment?

Comment (NRaeewal) [reply app/controllers/frequently_asked_questions_controller.rb]: Nope! Fixed

**Pull request merged**